shared/js/smart-screen/card_manager.js

```diff
@@ -806,14 +808,18 @@
     },
 
     isPinned: function cm_isPinned(options) {
-      var that = this;
-      return this._getPipedPromise('isPinned', function(resolve, reject) {
-        that._asyncSemaphore.wait(function() {
-          resolve(that._reloadCardList());
-        }, that);
-      }).then(function() {
-        return Promise.resolve(!!that.findCardFromCardList(options));
-      });
+      if (CardManager.ENABLE_MULTI_PIN) {
+        return Promise.resolve(false);
+      } else {
```

Comment:
I intended to make this feature configurable so the `else` is for the case `ENABLE_MULTI_PIN = false`.